controllers:
  - name: 'difftrack_mpc'
    type: 'difftrack_mpc/TrackingPidLocalPlanner'

controller:
  holonomic_robot: false

mbf_tolerance_check: true
dist_tolerance: 0.2
angle_tolerance: 0.5
